SW-data_analysis-20220111.do

```diff
@@ -92,11 +92,11 @@ tab depressed_2019 depressed_2020
 gen incidence_2020 = depressed_2019 if depressed_2019 < 1
 replace incidence_2020 = 1 if depressed_2019 == 0 & depressed_2020 == 1
 replace incidence_2020 = . if depressed_2019 ==1
+replace incidence_2020 = . if depressed_2020 ==. 

```

Comment:
I noticed that when ` depressed_2019 ==0 & depressed_2020 ==. ` , `incidence_2020` is coded as 0. Since `depressed_2020` is missing in this situation and you need it to determin the value of `incidence_2020`, I would suggest to recode those observation to missing.